widgets/src/api/pda/summary.js

```diff
@@ -0,0 +1,23 @@
+import makeRequest from 'api/makeRequest';
+import { DOMAINS, METHODS } from 'api/constants';
+import MOCK_SUMMARIES from 'mocks/summary/summaries';
+import MOCK_SUMMARY from 'mocks/summary/summary';
+
+export const getSummaries = async connection =>
```

Comment:
we need to call `getSummaries` in the config screen and let users pick a `summaryId` that they wan't the widget to work with

Reply:
now applied in `SummaryCardConfig` @gReis89